extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -32,7 +33,7 @@
   margin: auto !important;
   height: fit-content !important;
   top: 0 !important;
-  bottom: 0 !important;
+  bottom: 30px !important;
```

Comment:
`30px` could be a variable so it's easier to update / understand.
I see it used here, in chips-container and `tools-tab-preview`.